releasenotes/notes/add-openai-structured-outputs-906be78a984a1079.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    `OpenAIChatGenerator` and `AzureOpenAIChatGenerator` now support structured outputs by passing a `response_format` parameter.
```

Comment:
let's explain better: we support it via `generation_kwargs`. A short python example would not hurt.